File: app-kotlin/src/main/java/fudi/freddy/mokitosample/share/ShareContract.java
package fudi.freddy.mokitosample.share;

/**
 * Created by romelldominguez on 2/24/17.
 */

public interface ShareContract {

    interface ShareView{
        void reLoadList();
        void clearText();
        void loadText(String text);

    }

    interface ShareUserActions{
        void saveInput(String text); // Object
        void loadInput();
        void display(String dat);
    }
}

File: app-kotlin/src/main/java/fudi/freddy/mokitosample/share/ShareContract.kt
package fudi.freddy.mokitosample.share

/**
 * @version 1.0
 * @autor Romell Domínguez
 * @date 7/25/20
 */
interface ShareContract {
    interface ShareView {
        fun reLoadList()
        fun clearText()
        fun loadText(text: String?)
    }

    interface ShareUserActions {
        fun saveInput(text: String?) // Object
        fun loadInput()
        fun display(dat: String?)
    }
}